2996_reflection/reflection.md

```diff
@@ -3239,21 +3309,27 @@ Add a new subsection of [expr.prim]{.sref} following [expr.prim.req]{.sref}
 ::: addu
 **Expression Splicing   [expr.prim.splice]**
 
-FIXME: text for the template version.
-
 ```
 $splice-expression$:
    $splice-specifier$
-   template $splice-specifier$ < $template-argument-list$@~_opt_~@ >
 ```
 
-[#]{.pnum} The `$splice-specifier$` shall not designate an unnamed bit-field, a constructor or destructor, or a constructor template or destructor template.
+[#]{.pnum} The `$splice-specifier$` designates either
+
+* [#.#]{.pnum} a value or object,
+* [#.#]{.pnum} a variable,
+* [#.#]{.pnum} a structured binding,
+* [#.#]{.pnum} a function,
+* [#.#]{.pnum} an enumerator, or
+* [#.#]{.pnum} a non-static data member.
+
+The `$splice-specifier$` shall not designate an unnamed bit-field, a constructor, or a destructor.
```

Comment:
I think this should just go into "a function other than a constructor or destructor" and then unnamed bit-field can be a note after the last bullet that it's not a non-static data member.

Reply:
I like that, done.